docs/book/src/developer/providers/v1.1-to-v1.2.md

```diff
@@ -55,20 +55,83 @@ in ClusterAPI are kept in sync with the versions used by `sigs.k8s.io/controller
   [merge-strategy](https://kubernetes.io/docs/reference/using-api/server-side-apply/#merge-strategy).
   NOTE: the change will cause a rollout on existing clusters created with ClusterClass
 
-E.g. in CAPA
-
-```go
-// +optional
-Subnets Subnets `json:"subnets,omitempty"
-```
-Must be modified into:
-
-```go
-// +optional
-// +listType=map
-// +listMapKey=id
-Subnets Subnets `json:"subnets,omitempty"
-```
+  E.g. in CAPA
+
+  ```go
+  // +optional
+  Subnets Subnets `json:"subnets,omitempty"
+  ```
+  Must be modified into:
+
+  ```go
+  // +optional
+  // +listType=map
+  // +listMapKey=id
+  Subnets Subnets `json:"subnets,omitempty"
+  ```
+
+- [Server Side Apply](https://kubernetes.io/docs/reference/using-api/server-side-apply/) implementation in ClusterClass and managed topologies 
```

Comment:
Should we add an info that controller-runtime >= v0.12.3 is required?

Reply:
It could be useful to add controller runtime to the list on top of the page as well